packages/table-react/documentation/ExpandableTableExample.tsx

```diff
@@ -36,6 +37,7 @@ const ExpandableTableExample: VFC<ExampleComponentProps> = ({ boolValues, choice
                     <ExpandableTableRow
                         key={rowIndex}
                         clickable={markClickedRows ? { onClick: (e) => console.log(e), markClickedRows } : undefined}
+                        onOpenStateChange={(isOpen) => console.log({ isOpen })}
```

Comment:
Vi har et par eksemplarer av `onToggle` for liknende parameternavn. Ke du trur?

Reply:
Det tror jeg er mye mer passende. Sleit litt med å komme på et godt navn, så det der er resultatet av helt medioker navngiving :D